Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -82,5 +84,23 @@ YUI.add('ez-roleserversideview', function (Y) {
         _uiUnsetAssignRoleLoading: function (button) {
             button.removeClass('is-loading').set('disabled', false);
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Y.Node} button
+         */
+        _refreshAssignmentsTab: function (button) {
+            /**
+             * Fired when the view needs to be refreshed.
+             * @event refreshAssignmentsTab
+             * @param roleId {String} The ID of the role we are showing assignments for
+             */
+            this.fire('refreshAssignmentsTab', {
```

Comment:
> it should keep the selected tab and should reselect it after the refresh is done.

But some magic is required to make this happen, I think. I started with just the refresh, and added the tab handling when it turned out to be needed.